package.json

```diff
@@ -79,10 +80,17 @@
     "@commitlint/cli": "^17.6.7",
     "@commitlint/config-conventional": "^17.6.7",
     "@commitlint/format": "^17.4.4",
+    "@google/semantic-release-replace-plugin": "^1.2.0",
+    "@semantic-release/changelog": "^6.0.2",
+    "@semantic-release/exec": "^6.0.3",
+    "@semantic-release/git": "^10.0.1",
+    "@semantic-release/github": "https://registry.npmjs.org/@achingbrain/semantic-release-github/-/semantic-release-github-0.0.0.tgz",
```

Comment:
a bit curious on the approach here vs why not a "regular" version?

Reply:
From what Artur's told me we are using a fork with a github rate API bugfix :)

Reply:
That PR has finally been merged: https://github.com/semantic-release/github/pull/487 so just use the latest standard version here now

Reply:
In foundation we listed it as a dependency, because we used a custom version. Standard version is included with the semantic release itself (it's a default plugin), so you don't need to list it at all.